project#2- calculator-html, css, and js.html

```diff
@@ -0,0 +1,62 @@
+<html>
+    <head>
+        <script>
+            function insert(num){
+                document.form.textview.value=document.form.textview.value+num
+            }
+        </script>
+            <style>
+                *{
+                    margin:0;
+                    padding: 0;
+                }
+                .button{
+                    width:50;
+                    height: 50; 
+                    font-size: 25;
+                    margin: 2;
+                    cursor: pointer;
+                }
+                .textview{
+                    width: 217; 
+                    margin: 5;
+                    font-size: 25;
+                    padding: 5; 

```

Comment:
### Problem
Maybe I'm having issues here because my output _didn't come out the same_ as tutorial

Reply:
That wasn't the problem. 

*note to self: check personal project journal doc*